components/reddit/reddit.app.js

```diff
@@ -0,0 +1,59 @@
+const axios = require("axios");
+const get = require("lodash.get");
+
+module.exports = {
+  type: "app",
+  app: "reddit",
+  propDefinitions: {
+	  subreddit: {
```

Comment:
It looks like this indentation is off a bit, could you confirm auto-formatting is catching these changes in your editor?

Reply:
Yeah, settings on my editor.